python/lsst/analysis/tools/tasks/photometricCatalogMatch.py

```diff
@@ -84,6 +84,8 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         # the HSC ones do, this is a bit hacky but fixes this
         if bands[0].startswith("lsst") or "sim" in bands[0] or "smeared" in bands[0]:
             bands = self.config.filterNames
+        elif bands[0].startswith("monster"):
+            bands = [band[-1] for band in bands]
```

Comment:
Why the -1?  I don't fully follow this code (or why everything is so hacked up...)

Reply:
I added some more documentation, is this enough?

Reply:
Ah that helps.  This is ticketed to fix this correctly, right?  And fortunately we only have one character bands in the monster (maybe worth mentioning that we don't support NB in the monster).

Reply:
Added the ticket number to fix this